package.json

```
@@ -10,12 +10,12 @@
   ],
   "engines": {
     "node": ">=14",
-    "pnpm": ">=6.14.1 <7"
+    "pnpm": ">=6.14.1 <8"
```

We are still technically compative with pnpm 6, so I left it this way.

---

Since this is only for us, it might be better to say minimum v7 since we want every contributor to use v7 (the lockfile will be for v7)
```suggestion
    "pnpm": ">=7.3.0 <8"
```